pytorch_lightning/tuner/lr_finder.py

```diff
@@ -208,7 +209,7 @@ def lr_find(
     if update_attr:
         lr_attr_name = _determine_lr_attr_name(trainer, model)
 
-    save_path = os.path.join(trainer.default_root_dir, "lr_find_temp_model.ckpt")
+    save_path = os.path.join(trainer.default_root_dir, f"lr_find_temp_model_{uuid.uuid4()}.ckpt")
```

Comment:
Could you add a test ?

Reply:
we already have tests that ensure that model is reset correctly and since this is a completely random filename.. I am not sure what more to check in tests.